templates/common_header.php

```diff
@@ -92,15 +92,20 @@ function setDimension1(value) {
 
 <body <?php if (array_key_exists('body_class', $values)): ?>class="<?php echo $values['body_class']; ?>"<?php endif; ?>>
 
+<?php $DISPLAY_DONATION_BANNER = FALSE; ?>
+<?php if ($DISPLAY_DONATION_BANNER): ?>
+
 <div class="banner banner--donate">
     <div class="row">
         <div class="large-10 large-centered columns">
-            <strong>Help keep our sites running in 2018.</strong>
+            <strong>Help keep mySociety's sites running.</strong>
```

Comment:
Curly apostrophe please :)

Reply:
So given my other comment, this line would then be `<strong><?= BANNER_MESSAGE ?></strong>`